build/boilerplate/boilerplate.py

```diff
@@ -137,8 +137,8 @@ def get_regexs():
     regexs = {}
     # Search for "YEAR" which exists in the boilerplate, but shouldn't in the real thing
     regexs["year"] = re.compile( 'YEAR' )
-    # dates can be 2014, 2015 or 2016, company holder names can be anything
-    regexs["date"] = re.compile( '(2014|2015|2016|2017|2018|2019|2020)' )
+    # dates can be something in the 21st century
```

Comment:
LOL +1

Reply:
:) we'll see if cAdvisor lives into 22nd century haha